src/DynamoPackages/PackageLoader.cs

```diff
@@ -134,7 +134,14 @@ public void Load(Package package)
                 {
                     if (assem.IsNodeLibrary)
                     {
-                        OnRequestLoadNodeLibrary(assem.Assembly);
```

Comment:
By default all the .NET libraries are marked as IsNodeLibrary. Package author should have disabled it.